src/lib.rs

```diff
@@ -88,6 +88,11 @@ impl<E: Env> Ctap<E> {
         &mut self.hid
     }
 
+    #[cfg(feature = "fuzz")]
```

Comment:
@ia0 I don't really like exposing this here. But otherwise when using assembler individually (like in fuzz helper) we'll have to pass another env as parameter. But I think we should use the same env. Any better ways?

Reply:
I think this function makes sense even for normal tests so I would make it `std` instead of `fuzz`. The scenario I have in mind for tests is something like error injection:
- Create test env
- Create Ctap
- Do some stuff that use some part of the env is successful way
- Modify env to inject errors
- Do some stuff that use the same part of the env as before but this time should fail